Rakefile

```diff
@@ -5,6 +5,13 @@ require "rake/testtask"
 
 Rake::TestTask.new do |task|
   task.libs << "test"
+  task.test_files = Dir["test/**/*_test.rb"] - Dir["test/**/*_slow_test.rb"]
```

Comment:
Would `Dir["test/**/*_test.rb"]` not grab the `_slot_test.rb` files as well? And why the need for the 2nd task?